### ISSUE TITLE
When cloudcore not running，keadm reset failed

### ISSUE THREAD

**Issue by swartz-k** [kubeedge/kubeedge#2464]
**What happened**:
I have wrong certs，so I wanna use `keadm reset` to reset node，but it failed and have some errors when check `edge`

code located at `keadm/cmd/keadm/app/cmd/util/common.go:411` 
![image](https://user-images.githubusercontent.com/49771587/103137795-38e7f180-4707-11eb-9764-9093374c748e.png)

**What you expected to happen**:
skip error，just clean what exist

**How to reproduce it (as minimally and precisely as possible)**:
On cloudcore，`pkill cloudcore` and then run `keadm reset`

**Anything else we need to know?**:
I think we need update cloudcore to running as `systemd service` or when check command failed，just return `false，nil`

**Environment**:

<details><summary>CloudCore Version</summary>

```console
$ cloudcore --version

KubeEdge v1.5.0
```

**Comment by daixiang0** [comment]
/assign

Wait for #2254 merged, then optimize detect running logic.

**Comment by swartz-k** [comment]
> /assign
> 
> Wait for #2254 merged, then optimize detect running logic.

I read this PR，but not for my situation. Refer to keadm/cmd/keadm/app/cmd/util/common.go:411 
```
serviceRunning := fmt.Sprintf("systemctl list-unit-files | grep enabled | grep %s ", serviceName)
if err := NewCommand(serviceRunning).Exec(); err != nil {
    return false, err
}
```
I think need to be
```
serviceRunning := fmt.Sprintf("systemctl list-unit-files | grep enabled | grep %s ", serviceName)
if err := NewCommand(serviceRunning).Exec(); err != nil {
   // print error log or something
    return false, nil
}
```

**Comment by daixiang0** [comment]
You misunderstand it, i mean after #2254, i will post a new pr.

**Comment by Abirdcfly** [comment]
@daixiang0 can i just change 
`serviceRunning := fmt.Sprintf("systemctl list-unit-files | grep enabled | grep %s ", serviceName)` to `serviceRunning := fmt.Sprintf("systemctl list-unit-files | grep enabled | grep %s || true ", serviceName)` to fix this issue? 
the code just want to check if something return but forget grep will return exit code 1 if no line is matched. And that cause a error.

**Comment by daixiang0** [comment]
@Abirdcfly if so, you still need to check cmd output.
